tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -96,7 +215,11 @@ class MklCPUAllocator : public VisitableAllocator {
     }
 
     VLOG(1) << "MklCPUAllocator: Setting max_mem_bytes: " << max_mem_bytes;
-    allocator_ = new BFCAllocator(new MklSubAllocator, max_mem_bytes,
+
+    sub_allocator_ = new MklSubAllocator();
```

Comment:
Could you please add a comment saying that `BFCAllocator` (`large_size_allocator`) takes ownership of `sub_allocator_` and therefore we don't need to delete `sub_allocator_` in this class?